functions/src/adapter.ts

```diff
@@ -49,8 +49,11 @@ export function adaptDocument(
 ): Record<string, any> {
   const fields = getFieldsToIndex()
   const data = snapshot.data()
+  if (data && '_firestore_id' in data) {
```

Comment:
data can either be a `undefined/null` or an `object`? or can it be something else?

Reply:
No, it could be a `firestore.DocumentData` (an object) or `undefined`